plugins/chat/assets/javascripts/discourse/components/channels-list-public.gjs

```diff
@@ -75,35 +82,38 @@ export default class ChannelsListPublic extends Component {
           </div>
         {{/if}}
       </LinkTo>
-    {{/if}}
+    {{/if}} --}}
 
     {{#if this.displayPublicChannels}}
-      {{#if this.site.desktopView}}
-        <div class="chat-channel-divider public-channels-section">
-          {{#if this.inSidebar}}
-            <span
-              class="title-caret"
-              id="public-channels-caret"
-              role="button"
-              title="toggle nav list"
-              {{on "click" (fn this.toggleChannelSection "public-channels")}}
-              data-toggleable="public-channels"
+      {{! i guess we don't need this divider anymore?  }}
```

Comment:
I think it's probably safe to remove but we will likely want to keep the `LinkTo` buttons so that we can join channels and start new DMs, but maybe these can be added either at the top/bottom of the channel list.